packages/utils/typescript/lib/generators/utils.js

```diff
@@ -61,6 +60,9 @@ const saveDefinitionToFileSystem = async (dir, file, content) => {
  * @returns {Promise<string>}
  */
 const format = async (content) => {
+  // eslint-disable-next-line node/no-unsupported-features/es-syntax
+  const prettier = await import('prettier'); // ESM-only
```

Comment:
Did you test that the auto type gen still works well?

Reply:
I did now